news/2020-08-03-mappers-report-july.md

```diff
@@ -0,0 +1,178 @@
+
+### osu!taiko
+ 
+- **[Change the wording for "patterns faster than"](https://osu.ppy.sh/community/forums/topics/1097738)**: This change aims to clear up some ambiguity in the wording of a futsuu difficulty-specific rule.
+
+### osu!mania
+
+- **[Redefining the "stream" terminology](https://osu.ppy.sh/community/forums/topics/1079647])**: Another change mainly looking to make the terminology in the osu!mania Ranking Criteria as clear as possible. This change defines the length at which a pattern is considered a stream.
+- **[Change chord guidelines in normal difficulties](https://osu.ppy.sh/community/forums/topics/1067554)**: This is the only non-wording based change this month. This change, with regards to 4K and 7K normals difficulties, allows mappers to use chords during a long note hold as long as it's being played with the other hand.
+
+## Open Ranking Criteria Proposals
+
+As every month we keep you up to date with the changes to the Ranking Criteria. Additionally we list open proposals so you can give your feedback to them! Below a list of the new proposals since the last issue of the Mappers' Report.
+
+### All Modes
+
+- **[Treat official romanisation above RC standardisation](https://osu.ppy.sh/community/forums/topics/1095833)**
+- **[Avoid spoilers in maps](https://osu.ppy.sh/community/forums/topics/1083667)**
+- **[Clear metadata rules and add guidelines regarding mashups](https://osu.ppy.sh/community/forums/topics/1108054)**
+
+### osu!catch
+
+- **[Remove "clear dashes" guideline on Platter difficulties](https://osu.ppy.sh/community/forums/topics/1118458)**
+
+### osu!mania
+
+- **[Change some duplicated phrases in 7K hard guidelines](https://osu.ppy.sh/community/forums/topics/1112674)**
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+New Beatmap Nominators have arrived! As every month, we would like to show you who you can request your pending maps to. Just make sure to request them to the Beatmap Nominators of the right game mode. Below, a list of all additions and departures of this month!
+
+But before we get to them, we have some important announcements to make for the so-called "care package" which had its first test run some months ago in osu!taiko and a change regarding Beatmap Nominator applications and evaluations.
+
+### Activity lenience for returning Beatmap Nominators
+
+Have you ever been a Beatmap Nominator before and would like to make your return? Then we have great news! In the [February issue of the Mappers' Report](https://osu.ppy.sh/home/news/2020-02-28-mappers-report-february) we have mentioned that we applied a "care package" for osu!taiko that gave past Beatmap Nominators an easier way to make their return. Now the time has come that we will apply it for all game modes! And it starts with this announcement!
+Following rules apply for the activity lenience:
```

Comment:
```suggestion
The following requirements apply for specific activity leniency:
```
Suggesting "Requirements" over "Rules" as rules imply you need to hit both which doesn't make any sense, they're two separate points, and expanded the activity lenience to mention specifics so it confuses less the user